### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by Tool73** [authelia/authelia#8222] [Troubleshooting] [answered]
I am brand new to linux, Unraid, and github. I am trying to setup Authelia with NGINX Reverse Proxy. I finally got authelia to start but now in the logs I get: 
level=info msg="Access to 'https://MYCONTAINER.MYDOMAIN.org/' is forbidden to user '<anonymous>'" method=GET path=/api/verify remote_ip=172.70.42.126

It brings me up to a server 403. I'm assuming it won't let me because I'm unverified, but I can't verify if Authelia never shows up. If someone does answer, please answer like I'm a total tech idiot, because I am. Thanks ahead of time.

**Comment by james-d-elliott** [comment]
Show your access control rules.

**Comment by Tool73** [comment]
access_control:
  default_policy: deny
  rules:
    ## bypass rule
    - domain: 
        - "auth.domain.com"
      policy: bypass
    ## catch-all
    - domain:
        - "*.domain.com"
      subject: 
        - "group:admins"
      policy: one_factor

**Reply by james-d-elliott** [reply, marked as answer]
There's no rule for `mycontainer.mydomain.org` so the default policy of `deny` is applied.

**Comment by Tool73** [comment]
so I add all my domains on to here? Such as:

access_control:
default_policy: deny
rules:
## bypass rule
- domain:
- "auth.domain.com"
- pasta.mydomain.org
- overseerr.mydomain.org
- allcontainers.mydomain.org
policy: bypass
## catch-all
- domain:
- "*.domain.com"
subject:
- "group:admins"
policy: one_factor

Would this be correct? if so thanks

**Reply by james-d-elliott** [reply]
You'd remove the existing ones and replace them with your respective domains. But you seem to have the basic idea yes.

**Comment by Tool73** [comment]
awesome, thanks.
